src/js/program/program-controller.js

```diff
@@ -122,6 +122,8 @@ class ProgramController extends Events {
                     // We can synchronously reuse the current mediaController
                     // Reinitialize the mediaController with the new item, allowing a new playback session
                     mediaController.activeItem = playlistItem;
+                    mediaController.attached = true;
+                    mediaController.eventQueue.flush();
```

Comment:
Can these go in `_setActiveMedia`? It's called from a couple places and will always need to be attached.